### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN047.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#96] [General]
# https://blog.3place.cn/3place/lmyn/LMYN047.html

https://blog.3place.cn/3place/lmyn/LMYN047.html

**Comment by picklecai** [comment]
对于儿子来说，母亲虽然和蔼可亲，可是她会生下自己的竞争者。所以，对母亲好是一回事，毫不留情地杀死弟弟又是另一回事。

**Reply by picklecai** [reply]
虽然我不知道啥时候天帝释和底提成了母子关系了。

**Comment by picklecai** [comment]
苏摩达多的儿子呢，
名字就叫作迦拘蹉。 （1.46.16）

这两个名字好像出现过。

**Reply by picklecai** [reply]
迦拘蹉，是十车王的名字

**Comment by picklecai** [comment]
牟尼呀！你来到这首都。
我能够亲眼看到你来，
没有比这更大的幸福。

这句话真好！

**Comment by picklecai** [comment]
这全是我自己的过错。
神中之神波罗的杀戮者！
你丝毫也不能负责。 （1.46.2）

我的子宫倒了霉，
我想做件心爱的事情；

母性为什么是这样？？

**Comment by picklecai** [comment]
就这样母亲和儿子二人，
在苦行林里作了决定。

这母子关系，是不是从天帝释重新从底提肚子里出来而算的？所谓“佛母孔雀大明王”的母。
如果这样说，天帝释先杀死了未出生的，再自己成了“儿子”，那不就是鸠占鹊巢吗？
